cmd/init.go

```diff
@@ -31,6 +31,19 @@ var (
 	ports                 []string
 )
 
+/*
+**NOTE** to kedge devs:
+
+The structs are re-defined here because, if we use the structs defined in `types.go`
+it will clutter the output. Lot Some of upstream structs from OpenShift don't have
```

Comment:
`Lot Some of upstream` ?  Maybe just one of the `Lot` or `Some`

Reply:
hehe, fixed!